Differ.xcodeproj/project.pbxproj

```diff
@@ -424,7 +424,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_COMPILATION_MODE = wholemodule;
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = "1,2,4";
+				TARGETED_DEVICE_FAMILY = "1,2,3,4";
```

Comment:
I also prefer that the project configuration is kept out of the pbxproj - use the provided xcconfig files, please.